src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -314,11 +314,6 @@ def post(self, request: Request, organization) -> Response:
         """
         Add or invite a member to an organization.
         """
-        if not features.has("organizations:invite-members", organization, actor=request.user):
-            return Response(
-                {"organization": "Your organization is not allowed to invite members"}, status=403
-            )
```

Comment:
If we don't have this check anymore, how do we prevent saas developer plan organizations from creating members in excess of their plan limits?